<launch>
	<node name="test" pkg="drone_swarm" type="args_test.py" args="model:=drone1 test:=1" output="screen"/>
</launch>
